### ISSUE TITLE
whitespace check bug

### ISSUE THREAD

**Issue by zimbatm** [rodjek/puppet-lint#339]
I think the parser has some trouble parsing this horrible file:

```puppet
class foo {
  file{'/some/file': content    =>inline_template('
FOO=bar
    '),
    require =>Package[somepackage],
    }
  }
```
https://gist.github.com/zimbatm/c48ba94108aeb52a79a9

When running `puppet-lint` on it if fails with the following error:

```
`gems/puppet-lint-1.1.0/lib/puppet-lint/plugins/check_whitespace.rb:115:in `[]': no implicit conversion from nil to integer (TypeError)`
```

EDIT: removing the `require` part doesn't trigger the issue.

**Comment by fxlv** [comment]
thanks for the patch suggestion @zimbatm !

**Comment by rodjek** [comment]
This should be fixed as part of #409 (arrow_alignment check expecting an INDENT token at the start of each line)
